Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/model/ConceptEntry.java

```diff
@@ -0,0 +1,308 @@
+        this.broadens = broadens;
+    }
+    
+    public String getTypeId() {
+        return typeId;
+    }
+    
+    public void setTypeId(String typeId) {
+        this.typeId = typeId;
+    }
+    
+    public String getEqualTo() {
+        return equalTo;
+    }
+    
+    public void setEqualTo(String equal) {
+       this.equalTo = equal;
+    }
+    
+    public String getId() {
+        return id;
+    }
+    
+    public void setId(String id) {
+        this.id = id;
+    }
+    
+    public String getWordnetId() {
+        return wordnetId;
+    }
+    
+    public void setWordnetId(String wordnetId) {
+        this.wordnetId = wordnetId;
+    }
+    
+    public String getWord() {
+        return word;
+    }
+    
+    public void setWord(String word) {
+        this.word = word;
+    }
+       
+    public String getDescription() {
+        return description;
+    }
+    
+    public void setDescription(String description) {
+        this.description = description;
+    }   
+    
+    public String getPos() {
+        return pos;
+    }
+   
+    public void setPos(String pos) {
+        this.pos = pos;
+    }
+    
+    public String getConceptList() {
+        return conceptList;
+    }
+  
+    public void setConceptList(String conceptList) {
+        this.conceptList = conceptList;
+    }   
+    
+    public String getSimilarTo() {
+        return similarTo;
+    }
+   
+    public void setSimilarTo(String similar) {
+        this.similarTo = similar;
+    }
+   
+    public String getSynsetIds() {
+        return synsetIds;
+    }
+       
+    public void setSynsetIds(String synsetIds) {
+        this.synsetIds = synsetIds;
+    }
+  
+    public void setModified(String modified) {
+        this.modified = modified;
+    }
+   
+    public String getModified() {
+        return modified;
+    }
+   
+    public boolean isDeleted() {
+        return isDeleted;
+    }
+   
+    public void setDeleted(boolean isDeleted) {
+        this.isDeleted = isDeleted;
+    }
+   
+    public List<ChangeEvent> getChangeEvents() {
+         return this.changeEvents;
+    }
+   
+    public void addChangeEvent(ChangeEvent changeEvent) {
+        if(this.changeEvents == null) {
+            this.changeEvents = new ArrayList<>();
+        }
+        this.changeEvents.add(changeEvent);
+    }
+   
+    public String getModifiedUser() {
+        return modifiedUser;
+    }
+    
+    public void setModifiedUser(String modifiedUser) {
+        this.modifiedUser = modifiedUser;
+    }
+    
+    public String getMergedIds() {
+        return mergedIds;
+    }
+       
+    public void setMergedIds(String mergedIds) {
+        this.mergedIds = mergedIds;
+    }
+    
+    public Set<String> getAlternativeIds() {
+        if(this.alternativeIds != null) {
+            String[] ids = this.alternativeIds.split(",");
+            return new HashSet<>(Arrays.asList(ids));
+        }
+        return new HashSet<>();
```

Comment:
there are several places in the code that will call this method to add new ids. This won't work any longer, new ids will be added to the hashset but the hashset is not persisted. Why did you change that?